spec/fixtures/modules/error/plans/winrm_disconnect.pp

```diff
@@ -2,6 +2,7 @@
   TargetSpec $targets
 ) {
   $result = run_command('restart-service winrm', $targets, _catch_errors => true)
+  wait_until_available($targets, _catch_errors => true)
```

Comment:
I *think* this is still in line with what this test is for? Not sure how we got away with this for so long... Maybe this is actually exposing a real issue?